app/views/bhf/form/column/_file.html.haml

```diff
@@ -0,0 +1,3 @@
+= node f, field do
+  = f.file_field field.name, placeholder: t('bhf.helpers.form.file_field.select'), class: 'file-input'
+  %img#file-preview{width: '400', src: f.object.try(field.name).to_s}
```

Comment:
cleanedup: removed id because this partial is used multiple times, removed "try" because i hate trys. and "to_s" because no complicated column value structure is allowed anyways